code/modules/research/designs/weapon_designs.dm

```diff
@@ -266,3 +266,13 @@
 	materials = list(MAT_METAL = 2000, MAT_SILVER = 500)
 	build_path = /obj/item/weapon/suppressor
 	category = list("Weapons")
+
+/datum/design/gravitygun
+	name = "Zero-Point Energy Field Manipulator"
+	desc = "A multi-mode device that blasts Zero-Point Energy bolts that locally distort gravity."
+	id = "gravitygun"
+	req_tech = list("combat" = 5, "materials" = 6, "bluespace" = 5, "powerstorage" = 5)
```

Comment:
Personally I'd make it require magnets 6 but I guess this works too if you want to keep it.